internal/versions/versions.go

```diff
@@ -284,7 +284,7 @@ var VersionConfigs = map[ValidK8sVersion]KubernetesVersion{
 		CloudControllerManagerImageAWS: "registry.k8s.io/provider-aws/cloud-controller-manager:v1.26.0@sha256:fdeb61e3e42ecd9cca868d550ebdb88dd6341d9e91fcfa9a37e227dab2ad22cb", // renovate:container
 		// CloudControllerManagerImageGCP is the CCM image used on GCP.
 		// TODO: use newer "cloud-provider-gcp" from https://github.com/kubernetes/cloud-provider-gcp when newer releases are available.
-		CloudControllerManagerImageGCP: "ghcr.io/edgelesssys/cloud-provider-gcp:v25.2.0@sha256:86fa9d31ed0b3d0d8806f13d6e7debd3471028b2cb7cca3a876d8a31612a7ba5", // renovate:container
```

Comment:
This should be build on main and updated through renovate.

Reply:
I was not aware that our Renovate config is able to do so. Can you point me to the config part where it matches the K8s version to the image versions?

Reply:
https://github.com/edgelesssys/constellation/blob/main/renovate.json#L192 That's why there is the `// renovate:container` in the back. :wink:

Reply:
Quick sync with @katexochen: I have removed bump of version, so that we can use the image build from `main`. Will create a new PR once image build ran on `main`.